### DISCUSSION TITLE
Request support for Google's T5 v1.1 XXL encoder GGUF model

### DISCUSSION THREAD

**Discussion by RamonGuthrie** [lllyasviel/stable-diffusion-webui-forge#1333] [Ideas]
Forge support for Google's T5 v1.1 XXL encoder GGUF model

https://huggingface.co/city96/t5-v1_1-xxl-encoder-gguf

**Comment by ZeroCool22** [comment]
> Forge support for Google's T5 v1.1 XXL encoder GGUF model
> 
> https://huggingface.co/city96/t5-v1_1-xxl-encoder-gguf

Forgive my ignorance, this T5 is the thing that make the model to put more attention into what you write in the prompt, so it follows the indications better?

And if yes, how much good the google one is compared with the one we use right now?

**Reply by RamonGuthrie** [reply]
The T5 is from Google so it's the same model, used with SD3 and Flux

**Reply by MindQTRL** [reply]
Yes, that would probably be very useful for low VRAM users because of the lower size.
